example/lib/examples/map_controls_page.dart

```diff
@@ -0,0 +1,450 @@
+import 'package:flutter/material.dart';
+import 'package:yandex_mapkit_lite/yandex_mapkit_lite.dart';
+
+import 'package:yandex_mapkit_example/examples/widgets/control_button.dart';
+import 'package:yandex_mapkit_example/examples/widgets/map_page.dart';
+
+class MapControlsPage extends MapPage {
+  const MapControlsPage({Key? key}) : super('Map controls example', key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return _MapControlsExample();
+  }
+}
+
+class _MapControlsExample extends StatefulWidget {
+  @override
+  _MapControlsExampleState createState() => _MapControlsExampleState();
+}
+
+class _MapControlsExampleState extends State<_MapControlsExample> {
+  late YandexMapController controller;
+  final List<MapObject> mapObjects = [];
+
+  final MapObjectId targetMapObjectId = const MapObjectId('target_placemark');
+  static const Point _point = Point(latitude: 59.945933, longitude: 30.320045);
+  final animation = const MapAnimation(type: MapAnimationType.smooth, duration: 2.0);
+
+  bool tiltGesturesEnabled = true;
+  bool zoomGesturesEnabled = true;
+  bool rotateGesturesEnabled = true;
+  bool scrollGesturesEnabled = true;
+  bool nightModeEnabled = false;
+  bool fastTapEnabled = false;
+  bool mode2DEnabled = false;
+  ScreenRect? focusRect;
+  MapType mapType = MapType.vector;
+  int? poiLimit;
+  MapAlignment logoAlignment = const MapAlignment(
+    horizontal: HorizontalAlignment.left,
+    vertical: VerticalAlignment.bottom
+  );
+
+  final String style = '''
+    [
+      {
+        "tags": {
+          "all": ["landscape"]
+        },
+        "stylers": {
+          "color": "f00",
+          "saturation": 0.5,
+          "lightness": 0.5
+        }
+      }
+    ]
+  ''';
```

Comment:
I think here should be the link to docs, where can be founded all possible values of this config